lib/Map/MapboxVectorTileImageryProvider.ts

```diff
@@ -49,8 +49,7 @@ interface MapboxVectorTileImageryProviderOptions {
   featureInfoFunc?: (feature: any) => ImageryLayerFeatureInfo | undefined;
 }
 
-export default class MapboxVectorTileImageryProvider
-  implements ImageryProvider {
+export default class MapboxVectorTileImageryProvider extends ImageryProvider {
```

Comment:
Didn't we talk to @kring about this and it should still be `implements`, with implementation of the missing methods?

Reply:
Yup, my bad! Forgot to fix this after what we discussed